src/index.js

```diff
@@ -86,7 +86,7 @@ class ReduxSagaFirebase {
   projectId () {
     if (this._projectId) return this._projectId
 
-    const regex = /^([a-z0-9-]+?)(?:-[a-z0-9]{5})?\.firebaseapp\.com$/
+    const regex = /^([a-z0-9-]+?(-[a-z0-9]{5})*)?\.firebaseapp\.com$/
```

Comment:
Do we need an edge case for the last 5 digits? It seems like `^([a-z0-9-]+)\.firebaseapp\.com$` would be enough to solve this (and it's easier to read 🙂).

Reply:
Hi @n6g7! I don't really know if they have a different structure for the last five digits. It seems like we don't need it anymore... about the regex it looks better that way!! 👍

Reply:
Cool I'll merge and update the regex myself. :)